### ISSUE TITLE
Wiki to create my own Rock64 img

### ISSUE THREAD

**Issue by jonnyfoxy** [ayufan-rock64/linux-build#230]
Hi, 

I would like to create my own Rock64 image build from scratch. For example, using the [Ubuntu Arm Server ISO](https://www.ubuntu.com/download/server/arm). What would be the steps to accomplish such a task?

Thanks

**Comment by jonnyfoxy** [comment]
Any thoughts will be appreciated.

**Comment by evilbunny2008** [comment]
I wrote up a cheat sheet to customise an image

https://github.com/evilbunny2008/rock64-webgui/#manual-install

**Comment by jonnyfoxy** [comment]
Your example is pretty amazing!!!
and will help me modifying images, regardless for what Linux img I want to create.

but I am also looking into how to make a clean install from scratch. for example using the [Ubuntu server ISO](https://www.ubuntu.com/download/server/arm) and creating a .img file from it.

maybe using [qemu](https://www.qemu.org/) and creating an img from it.
Other option, if possible directly on the rock64 using 2 USB devices. one being the installer and the other will be the storage where the OS will be installed on.
or if you have more ideas that can work some magic, would be great to learn.

Thanks

**Comment by evilbunny2008** [comment]
I don't think you can use ubuntu images like that, ayufan's builds all have patches, even for the mainline kernels.

**Comment by jonnyfoxy** [comment]
Does he have a guide on creating them?
It would be awesome to know how to do it.

**Comment by mrfixit2001** [comment]
@jonnyfox - easiest way to make patches locally is by comparing two files, one original and one with the changes you want the patch to include. Type "diff -Naur (original file) (updated file) > mypatch.patch"
